bin/server.js

```diff
@@ -29,6 +29,7 @@ if (process.env.S3_SERVER_SECURE_KEY_FILE || argv.secureKey) {
 }
 
 const prefix = argv.prefix || process.env.S3_KEY_PREFIX || "";
+const overrideContentType = process.env.OVERRIDE_CONTENT_TYPE || null
```

Comment:
shouldn't we receive this parameter via cli too? like how it is done in the other parameters